tests/trace_service.py

```diff
@@ -14,16 +14,16 @@ def Export(self, request, context):
         self.batches.append(request.resource_spans)
         return trace_service_pb2.ExportTracePartialSuccess()
 
-    def get_span(self):
+    def get_batch(self):
         for _ in range(10):
             if len(self.batches):
                 break
             time.sleep(0.001)
+        assert len(self.batches) == 1
+        return self.batches.pop()
```

Comment:
I think, we should go a step further and return self.batches[0][0], i.e.
```suggestion
        assert len(self.batches) == 1
        assert len(self.batches[0]) == 1
        return self.batches.pop()[0]
```